package-lock.json

```diff
@@ -8,7 +8,7 @@
       "name": "scripting-starter",
       "version": "0.1.0",
       "dependencies": {
-        "@minecraft/server": "1.4.0-beta.1.20.10-stable",
+        "@minecraft/server": "^1.6.0-beta.1.20.30-stable",
```

Comment:
Should we try to target exact versions for `@minecraft/server`? Given how frequent Microsoft is updating, this may be a rare situation that we are okay with this not being an exact version 🤔

Reply:
Since this project utilizes the API we can keep running whatever version of the minecraft/server as a dependency until they change the underlying API calls we are using. We can hold any exact version as long as they don't change the call up.